docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -0,0 +1,308 @@
+	+---------+-------------------------------------------------+------+
+	| husavik | 34:3b:58:ba:ec:95:f5:17:17:df:04:38:11:89:e6:3d | ssh  |
+	+---------+-------------------------------------------------+------+
+	```
+	
+	...and set to the `KEYPAIR` variable the name of the keypair 
+	you want:
+	
+	```bash
+	KEYPAIR="husavik" # again, this is just an example
+	```
+	
+	Finally, decide on a name for your new Kubernetes cluster:
+	
+	```bash
+	CLUSTER_NAME="bangor" # just remembered a famous storyteller
+	```
+	
+	With everything in place, go ahead and create your new 
+	Kubernetes cluster:
+	
+	```bash
+	openstack coe cluster create \
+		--cluster-template $CLUSTER_TMPL \
+		--keypair husavik
+		bangor
+	```
+	If everything went well with your request for a new cluster, 
+	on your terminal, you would see a message like the following:
+	 
+	 ```plain
+	 Request to create cluster e0df8c62-c6f6-4c7d-b67e-33e3606e9ab6 
+	accepted
+	 ```
+	
+	The cluster creation process takes some time to complete, and 
+	while you are waiting, you can check if everything is progressing 
+	smoothly:
+	
+	```bash
+	openstack coe cluster list -c status
+	```
+	
+	If everything is going well, the message you will get will be 
+	`CREATE_IN_PROGRESS`. When Magnum has finished creating the cluster, 
+	the message will be `CREATE_COMPLETE`.
+
+## Viewing the Kubernetes cluster
+
+After the Kubernetes cluster is ready, you may at any time view it and 
+get detailed information about it.
+
+=== "{{gui}}"
+	In the left vertical pane of the {{gui}}, click through 
+	_Kubernetes_, _Magnum_, and _Clusters_. In the central pane on the 
+	right, you will then see all your Kubernetes clusters in every region.
+	
+	![All k8s clusters](assets/new-k8s/shot-04.png)
+	
+	Click on the three-dot icon on the right of the cluster you 
+	want to inspect, and select _View details_.
+
+	![Cluster details](assets/new-k8s/shot-05.png)
+=== "OpenStack CLI"
+	To list all available Kubernetes clusters, just type:
+	
+	```bash
+	openstack coe cluster list
+	```
+
+	```plain
+	+--------------------------------------+--------+---------+------------+--------------+-----------------+---------------+
+	| uuid                                 | name   | keypair | node_count | master_count | status          | health_status |
+	+--------------------------------------+--------+---------+------------+--------------+-----------------+---------------+
+	| e0df8c62-c6f6-4c7d-b67e-33e3606e9ab6 | bangor | husavik |          1 |            1 | CREATE_COMPLETE | HEALTHY       |
+	+--------------------------------------+--------+---------+------------+--------------+-----------------+---------------+
+	```
+
+	For many more details on a specific cluster, note its name and 
+	run a command like this:
+	
+	```bash
+	openstack coe cluster show bangor
+	```
+
+	```plain
+	+----------------------+---------------------------------------------------------------------------------------------------------+
```

Comment:
Here, too, we're in excess of 100 characters.

Reply:
In my RC. Forever.